snuba/migrations/groups.py

```diff
@@ -162,6 +165,11 @@ def __init__(
         storage_sets_keys={StorageSetKey.GROUP_ATTRIBUTES},
         readiness_state=ReadinessState.PARTIAL,
     ),
+    MigrationGroup.METRICS_SUMMARIES: _MigrationGroup(
+        loader=MetricsSummariesLoader(),
+        storage_sets_keys={StorageSetKey.METRICS_SUMMARIES},
+        readiness_state=ReadinessState.PARTIAL,
```

Comment:
One thing to note, if we're setting this new migration group to `partial` immediately, this requires the storage_key for metrics summary to be mapped to an existing clickhouse cluster in both SaaS and S4S. We should check that is true before this gets merged.

Reply:
Good point. On a similar note, do you know if there is a migration state which enables migrations only on S4S but not SaaS? I believe that is what would be needed here to begin with.

Reply:
Enoch mentioned this offline, but we don't have a state that is specific to S4S yet. We'll have to add that before this can be merged.

Reply:
https://github.com/getsentry/snuba/pull/5172

Reply:
Based on what I understand, is the next step adding the new state here in S4S https://github.com/getsentry/ops/blob/b484e0f8d5f41f9ecab61997d70de9530f3fd44b/single-tenant/k8s/customers/s4s.yaml#L403-L406?

Reply:
I see experimental is already added in s4s. Thanks @evanh